p2p/p2p.proto

```diff
@@ -45,8 +45,37 @@ enum AccessType {
   ACCESS_TYPE_ACCESS_TOKEN_VERIFICATION = 3;
 }
 
+message AccessToken {
+  // The identifier is an internal 256bit word
+  bytes identifier = 1;
+  // The identity granting access to the document
+  bytes granter = 7;
+  // The identity being granted access to the document
+  bytes grantee = 8;
+  // Role identifier is the identifier on the read rule that this token should be mapped to
+  bytes role_identifier = 9;
+  // Original identifier of the document
+  bytes document_identifier = 2;
+  // TODO: if set, access will be limited to the specific version
```

Comment:
I'd leave the comments out of the PR for now. Who knows when we'll implement the below features.